core/scratch_block_comment.js

```diff
@@ -415,6 +456,33 @@ Blockly.ScratchBlockComment.prototype.getHeightWidth = function() {
   return {height: this.height_, width: this.width_};
 };
 
+/**
+ * Returns the coordinates of a bounding box describing the dimensions of this
+ * comment.
+ * Coordinate system: workspace coordinates.
+ * @return {!{topLeft: goog.math.Coordinate, bottomRight: goog.math.Coordinate}}
+ *    Object with top left and bottom right coordinates of the bounding box.
+ * @package
+ */
+Blockly.ScratchBlockComment.prototype.getBoundingRectangle = function() {
+  var commentXY = this.getXY();
+  var commentBounds = this.getHeightWidth();
+  var topLeft;
+  var bottomRight;
+  if (this.workspace.RTL) {
+    // TODO for some reason this doesn't work with workspace scroll in RTL
```

Comment:
Make an issue for this and add the number here.

Reply:
#1562